src/shape/Sector.tsx

```diff
@@ -246,6 +246,11 @@ export class Sector extends PureComponent<Props> {
       path = getSectorPath({ cx, cy, innerRadius, outerRadius, startAngle, endAngle });
     }
 
-    return <path {...filterProps(this.props, true)} className={layerClass} d={path} role="img" />;
+    return (
+      <>
+        <path {...filterProps(this.props, true)} className={layerClass} d={path} />
+        <title>{`${(this.props as any).name} ${(this.props as any).value}`}</title>
```

Comment:
We can't really do this. We want to avoid adding more `any's` than are already in the repo. We should find a mechanism to add these as props where we can have type safety. 

What is the impact to `Pie`?

Sorry I didn't call this out earlier